packages/vue/src/tooltip/tooltip.tsx

```diff
@@ -68,85 +68,78 @@ export default defineComponent({
 
     useMounted([updateLocaltion, removePoperInstance])
 
-    const preventHandler = (e: Event) => {
-      e.stopPropagation()
-      e.stopImmediatePropagation()
-    }
-
-    // render arrow Node
-    const renderArrowIcon = () => <span class={bem('arrow', props.type)} />
-
-    /**
-     * render content Node
-     */
     const renderContent = () => {
       const contentSlot = slots['content']
-      const { visibleArrow, content, portalClass, type } = props
-      return (
-        <Teleport
-          teleport={teleport.value}
-          scroll={false}
-          popupClass={bem('content', type) + ' ' + portalClass}
-          onPopupClick={preventHandler}
-          onMouseenter={() => mouseEventHandler(true)}
-          onMouseleave={() => mouseEventHandler(false)}
-          show={show.value}
-          ref={contentRef}
-        >
-          {visibleArrow && renderArrowIcon()}
+      const { visibleArrow, content, portalClass, type, teleport } = props
 
-          <div class={bem('inner')}>{contentSlot ? contentSlot() : content}</div>
+      return (
+        <Teleport to={teleport}>
+          <div
+            class={withClassName(bem('content', type), portalClass)}
+            ref={tooltipContentRef}
+            v-show={visible.value}
+            onMouseenter={() => mouseEventHandler(true)}
+            onMouseleave={() => mouseEventHandler(false)}
```

Comment:
I think can use `visible.value&&<element />` way to render the reference. Use the `v-show`. Only add `display:none` for each `Tooltip Component` may cause to render a large number of elements and page blocking. 
See the [popper.js](https://popper.js.org/docs/v2/modifiers/).